stages/org.osbuild.authselect

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Select system identity and authentication sources with authselect.
+
+Sets system identity and authentication sources.
+
+The stage calls `authselect select` to set authselect profile to 'profile_id'.
+Optionally a list of profile features to enable may be provided using 'features'
+option. The list of available profile features can be obtained by running
+`authselect list-features <profile_id>`.
+
+Notes:
+  - Requires 'chroot' in the buildroot.
+  - Runs the 'authselect' binary from the image in the chroot.
+"""
+
+
+import subprocess
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["profile_id"],
+"description": "Select system identity and authentication sources.",
+"properties": {
+  "profile_id": {
+    "type": "string",
+    "description": "Desired authselect profile to activate."
+  },
+  "features": {
+    "type": "array",
+    "description": "Features of the selected profile to activate.",
+    "minItems": 1,
+    "items": {
+      "type": "string"
+    }
+  }
+}
+"""
+
+
+def main(tree, options):
+    profile_id = options["profile_id"]
```

Comment:
This is an opaque string, or selected from a well-known list?

Reply:
I took the name from authselect man page:
```
select profile_id [features] [-f, --force] [-q, --quiet] [-b] [--backup=NAME]
```

Reply:
Right, I looked up the man page. It seems like the supported profiles/features are dynamic, rather than hard-coded in authselect, so I guess we should not try to ship an allow-list either (which usually is my preference, but I guess it will make our lives unnecessarily complicated).

Reply:
oh, you probably meant the value that can be passed here, right? The list is not fixed and can be different per distro (and additional profiles could be probably installed via custom RPM). Valid values are the ones returned by `authselect list`. Checking the value against a fixed list of values is not viable here.

Reply:
Yup, that's exactly what I meant :)